packages/components/inputs/money-input/src/money-input.js

```diff
@@ -89,20 +89,19 @@ const createCurrencySelectStyles = ({
       borderRight: '0',
       minWidth: '72px',
       borderColor: (() => {
-        if (isDisabled) return vars.borderColorForInputWhenDisabled;
-        if (isReadOnly) return vars.borderColorForInputWhenReadonly;
+        if (isDisabled)
+          return `${vars.borderColorForInputWhenDisabled} !important`;
```

Comment:
I'm using `!important` here because when the input is disabled, no other states matter anymore (focus, hover, etc) so it's simpler to do this instead of going through each of those state selectors just to set the default. Also because react-select has pre-defined styles and states.